man/systemd.unit.xml

```diff
@@ -1031,6 +1033,15 @@
         whether a certain path exists, is a regular file and marked
         executable.</para>
 
+        <para><varname>ConditionUID=</varname> takes a numeric
+        <literal>UID</literal>. This condition may be used to check
+        wether the service manager is running as the given user id.
+        This option is only meaningful for the user managers.</para>
+
+        <para><varname>ConditionGID=</varname> is similar
+        to <varname>ConditionUID=</varname> but verifies that the
+        service manager is in the specified group.</para>
```

Comment:
Hmm. I just noticed `in_gid` checks both real and effective gid. I think it would be confusing to have ConditionUID ignore effective UID, and ConditionGID take into account the effective GID (even though in practice it shouldn't matter much, because AFAICT, by the time the conditions are tested the service managers do not perform any uid changes).
So, what would be best? Make ConditionGID ignore the effective group, or making ConditionUID take it into account?